fftw-wisdom dependency in configure

Updating from 7b96d7c..a5a4d2b causes configure to break, citing an inability to find fftw-wisdom. 

> checking for fftw-wisdom... no
> configure: error: FFTW3 requested but fftw-wisdom not found

@d7919 and I discussed this on the general Slack discussion.

--- a/configure
+++ b/configure
@@ -6166,7 +6166,7 @@ fi
 
   if test "x$fftw_path" != "xno"; then :
 
-  fftw_wisdom0=`$as_dirname -- "$fftw_path" ||
+    fftw_wisdom0=`$as_dirname -- "$fftw_path" ||
 $as_expr X"$fftw_path" : 'X\(.*[^/]\)//*[^/][^/]*/*$' \| \
 	 X"$fftw_path" : 'X\(//\)[^/]' \| \
 	 X"$fftw_path" : 'X\(//\)$' \| \
@@ -6189,7 +6189,7 @@ $as_echo X"$fftw_path" |
 	    q
 	  }
 	  s/.*/./; q'`
-  fftw_wisdom=`$as_dirname -- "$fftw_wisdom0" ||
+    fftw_wisdom=`$as_dirname -- "$fftw_wisdom0" ||
 $as_expr X"$fftw_wisdom0" : 'X\(.*[^/]\)//*[^/][^/]*/*$' \| \
 	 X"$fftw_wisdom0" : 'X\(//\)[^/]' \| \
 	 X"$fftw_wisdom0" : 'X\(//\)$' \| \
@@ -6212,7 +6212,12 @@ $as_echo X"$fftw_wisdom0" |
 	    q
 	  }
 	  s/.*/./; q'`
-  with_fftw="$with_fftw $fftw_wisdom"
+    with_fftw="$with_fftw $fftw_wisdom"
+
+else
+  { $as_echo "$as_me:${as_lineno-$LINENO}: FFTW3 requested but fftw-wisdom not found" >&5
+$as_echo "$as_me: FFTW3 requested but fftw-wisdom not found" >&6;}
+fi
 
 
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking for fftw3.h" >&5
@@ -6462,9 +6467,6 @@ ac_compiler_gnu=$ac_cv_cxx_compiler_gnu
   CXXFLAGS="$CXXFLAGS -DBOUT_HAS_FFTW"
   HAS_FFTW="yes"
 
-else
-  as_fn_error $? "FFTW3 requested but fftw-wisdom not found" "$LINENO" 5
-fi
 else
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: Configuring without FFTW3 is not recommended" >&5

--- a/configure.ac
+++ b/configure.ac
@@ -396,9 +396,10 @@ AS_IF([test "x$with_fftw" != "xno"], [
 AC_PATH_PROG([fftw_path], [fftw-wisdom], [no], [$with_fftw$PATH_SEPARATOR$PATH])
 
   AS_IF([test "x$fftw_path" != "xno"], [
-  fftw_wisdom0=`AS_DIRNAME(["$fftw_path"])`
-  fftw_wisdom=`AS_DIRNAME(["$fftw_wisdom0"])`
-  with_fftw="$with_fftw $fftw_wisdom"
+    fftw_wisdom0=`AS_DIRNAME(["$fftw_path"])`
+    fftw_wisdom=`AS_DIRNAME(["$fftw_wisdom0"])`
+    with_fftw="$with_fftw $fftw_wisdom"
+  ], AC_MSG_NOTICE([FFTW3 requested but fftw-wisdom not found]))
 
   BOUT_ADDPATH_CHECK_HEADER(fftw3.h, ,AC_MSG_ERROR([FFTW3 requested but header not found]), $with_fftw)
   BOUT_ADDPATH_CHECK_LIB(fftw3, fftw_plan_dft_r2c_1d, ,AC_MSG_ERROR([FFTW3 requested but library not found]), $with_fftw)
@@ -407,7 +408,7 @@ AC_PATH_PROG([fftw_path], [fftw-wisdom], [no], [$with_fftw$PATH_SEPARATOR$PATH])
   #add our define to CXXFLAGS
   CXXFLAGS="$CXXFLAGS -DBOUT_HAS_FFTW"
   HAS_FFTW="yes"
-], AC_MSG_ERROR([FFTW3 requested but fftw-wisdom not found]))],
+],
 [
 AC_MSG_NOTICE([Configuring without FFTW3 is not recommended])
 HAS_FFTW="no"

